### PR TITLE
modify css style

### PR DESCRIPTION
**Pull request by membphis**
move it

### CODE REVIEW

File: v2/css/main.less [hunk 1]
```diff
@@ -121,6 +121,7 @@ header[role="header"] {
 			span { color: @accent2; font-weight: 300; }
  		}
  		small { display: block; font-size: 13px; color: @bg; }
+        img {margin-top: -6px; margin-left: -20px; margin-right: 10px;}
```
Comment (agentzh) [inline v2/css/main.less]: Indentation problem?

**Comment by agentzh** [comment]
@membphis I think you need to generate the main.css from main.less as well :)

File: v2/css/main.less (hunk 1 above)
Comment (membphis) [inline v2/css/main.less]: http://openresty.org/en/

Comment (membphis) [inline v2/css/main.less]: http://openresty.org/en/

the logo position is not ok.  the css file is not upgraded.

**Comment by membphis** [comment]
generate the new main.css file, pls check it. ^_^

File: v2/css/main.less (hunk 1 above)
Comment (agentzh) [inline v2/css/main.less]: @membphis it was a CDN cache purge issue and only visible in China mainland. Alas. Just purges the CSS file again and it should work now.

BTW, we need a space after `{` and also a space before `}` in your CSS line.

**Comment by agentzh** [comment]
@membphis Thanks for your fix! Merged with some minor edits and also rebased to the new master.

**Pull request closed without merging**